mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -136,15 +136,25 @@ def create(self, vals):
         """Creation of Action."""
         Sequence = self.env['ir.sequence']
         vals['reference'] = Sequence.next_by_code('mgmtsystem.action')
-        action = super(MgmtSystemAction, self).create(vals)
-        self.send_mail_for_action(action)
+        action = super(MgmtsystemAction, self).create(vals)
         return action
 
     @api.multi
+    def _track_template(self, tracking):
+        self.ensure_one()
+        res = super(MgmtsystemAction, self)._track_template(tracking)
+        changes, dummy = tracking[self.id]
+        if 'stage_id' in changes and self.stage_id.mail_template_id:
+            res['stage_id'] = (
+                self.stage_id.mail_template_id,
+                {'composition_mode': 'mass_mail'})
+        return res
+
+    @api.multi
     def write(self, vals):
         """Update user data."""
         if vals.get('stage_id'):
-            stage_new = self._get_stage_new()
```

Comment:
Hi @grindtildeath 
I changed the methods name because stage_new() implies creating a new stage, the name does not imply on the methods intention to retrive the default stage from the database.

So if possible i would like to stay consistent to the odoo core modules with stage_default or something like get_default_stage_id so that the methods name at least contains the word default in it.
I chose _default_stage since this was the shortest method name fullfillig the requirement of not beein missleading.

/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/tests/test_lead2opportunity.py:            'stage_id': default_stage_id
/opt/odoo/10/srv/addons/crm/models/crm_lead.py:        stage_id = self._default_stage_id()
/opt/odoo/10/srv/addons/crm/models/crm_lead.py:    def _default_stage_id(self):
/opt/odoo/10/srv/addons/crm/models/crm_lead.py:        group_expand='_read_group_stage_ids', default=lambda self: self._default_stage_id())
/opt/odoo/10/srv/addons/project_issue/models/project_issue.py:    def _get_default_stage_id(self):
/opt/odoo/10/srv/addons/project_issue/models/project_issue.py:                               default=_get_default_stage_id)
/opt/odoo/10/srv/addons/survey/models/survey.py:    def _default_stage(self):
/opt/odoo/10/srv/addons/survey/models/survey.py:    stage_id = fields.Many2one('survey.stage', string="Stage", default=_default_stage,
/opt/odoo/10/srv/addons/project/models/project.py:    def _get_default_stage_id(self):
/opt/odoo/10/srv/addons/project/models/project.py:        default=_get_default_stage_id, group_expand='_read_group_stage_ids',
/opt/odoo/10/srv/addons/note/models/note.py:    def _get_default_stage_id(self):
/opt/odoo/10/srv/addons/note/models/note.py:        string='Stages of Users',  default=_get_default_stage_id)